libbpf-tools/funclatency.c

```diff
@@ -292,6 +293,11 @@ int main(int argc, char **argv)
 		return 1;
 	}
 
+      if (!check_kern_array_mmap()) {
```

Comment:
It's simpler to do

```
if (!obj->rodata) { warn(...) ; }
```

few lines below after `funclatency_bpf__load()` call.

Reply:
Should be `obj->bss` ?

Reply:
doesn't matter, obj->rodata, obj->data, and obj->bss are all set to NULL if kernel doesn't support memory-mapping of array map

Reply:
adjusted.